website/docs/d/rds_engine_version.markdown

```diff
@@ -27,6 +37,7 @@ The following arguments are supported:
 * `parameter_group_family` - (Optional) Name of a specific DB parameter group family. Examples of parameter group families are `mysql8.0`, `mariadb10.4`, and `postgres12`.
 * `preferred_versions` - (Optional) Ordered list of preferred engine versions. The first match in this list will be returned. If no preferred matches are found and the original search returned more than one result, an error is returned. If both the `version` and `preferred_versions` arguments are not configured, the data source will return the default version for the engine.
 * `version` - (Optional) Version of the DB engine. For example, `5.7.22`, `10.1.34`, and `12.3`. If both the `version` and `preferred_versions` arguments are not configured, the data source will return the default version for the engine.
+* `engine-mode` - (Optional) Accepts DB engine modes for Aurora. The results list only includes information about the DB engine versions for these engine modes. Valid DB engine modes are the following: `global`, `multimaster`, `parallelquery`, `provisioned` and `serverless`.
```

Comment:
Since you're adding a value of `filter` to the schema, the documentation should align to that change as well. You can also reference the AWS documentation instead of listing the values explicitly: https://docs.aws.amazon.com/cli/latest/reference/rds/describe-db-engine-versions.html

```suggestion
* `filter` - (Optional) One or more name/value pairs to use as filters. See the [API documentation](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DescribeDBEngineVersions.html) for valid name/value pairs.
```